### ISSUE TITLE
Update Projects Editor based on new mocks

### ISSUE THREAD

**Issue by plante-msft** [JaneaSystems/PowerToys-DevProjects#67]
Tons of feedback from the team on the Projects Editor, so I've come up with some new mocks with additional features including:
- New app removal/add back UX
- Changed list view to include expandable/collapsible sections per app which houses CLI arguments, window size modifiers and "launch as admin" settings
- H2 view below app name to see glanceable admin and args values without expanding
- Launch & Edit and Revert buttons added

![Image](https://github.com/user-attachments/assets/25bfe463-d000-41dd-8c7a-b3f0194eab93)

![Image](https://github.com/user-attachments/assets/c7e85910-2b95-46af-b3b3-42ebb4cebb1d)

cc @crutkas @hamza-usmani

**Comment by SeraphimaZykova** [comment]
To clarify, do we need to revert changes related to app selection, as in this issue https://github.com/JaneaSystems/PowerToys-DevProjects/issues/50?

**Comment by plante-msft** [comment]
Yes, apologies. I spoke with a lot of folks who felt the checkboxes were somewhat confusing (since normally checkboxes have a series of selectable actions tied to them), so changing to an explicit "remove/add" structure makes more sense.
